explainability-core/src/main/java/org/kie/trustyai/explainability/metrics/FairnessMetrics.java

```diff
@@ -136,6 +137,64 @@ public static double groupStatisticalParityDifference(Dataframe priviledged, Dat
         return probabilityUnprivileged - probabilityPrivileged;
     }
 
+    /**
+     * Produce a general explanation of Group Statistical Parity Difference
+     */
+    public static String defineGroupStatisticalParityDifference() {
```

Comment:
Minor and optional comment, I would prefer to keep `FairnessMetrics` class for the algorithm while similar descriptions/explanations might be moved to a `FairnessMetricsUtils` class under [`utils`](https://github.com/trustyai-explainability/trustyai-explainability/tree/main/explainability-core/src/main/java/org/kie/trustyai/explainability/utils) package.

Reply:
Yep, sounds good